book_maker/loader/pdf_loader.py

```
@@ -0,0 +1,244 @@
+        is_test=False,
+        test_num=5,
+        prompt_config=None,
+        single_translate=False,
+        context_flag=False,
+        context_paragraph_limit=0,
+        temperature=1.0,
+        source_lang="auto",
+        parallel_workers=1,
+    ) -> None:
+        if fitz is None:
+            raise Exception("PyMuPDF (fitz) is required to use PDF loader")
```

Ruff (0.15.12)

[warning] Create your own exception

(TRY002)